pkg/tfbridge/provider_test.go

```diff
@@ -331,5 +369,15 @@ func TestProviderPreviewV2(t *testing.T) {
 			Schema: &ResourceInfo{Tok: "ExampleResource"},
 		},
 	}
+
 	testProviderPreview(t, provider)
+
+	failures := testCheckFailures(t, provider)
+	sort.SliceStable(failures, func(i, j int) bool { return failures[i].Reason < failures[j].Reason })
+	assert.Equal(t, "ConflictsWith: \"conflicting_property\": conflicts with conflicting_property2", failures[0].Reason)
+	assert.Equal(t, "", failures[0].Property)
+	assert.Equal(t, "ConflictsWith: \"conflicting_property2\": conflicts with conflicting_property", failures[1].Reason)
+	assert.Equal(t, "", failures[1].Property)
+	assert.Equal(t, "Required attribute is not set", failures[2].Reason)
+	assert.Equal(t, "", failures[2].Property)
```

Comment:
Slightly related to the above - it feels wrong that we don't populate `Property` on any of these failures - but it appears we've never done that for any errors coming from Check - so this is a separate issue.  It is not immediately clear to me whether we could reliably map an `AttributePath` into a `Property`.

Reply:
I agree that it feels strange that we don't populate that information. If we can, we should, though I'm not entirely certain that anything would use it at the moment.